lighthouse-plugin-ad-speed-insights/audits/script-injected-tags.js

```diff
@@ -0,0 +1,121 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const NetworkRecords = require('lighthouse/lighthouse-core/computed/network-records');
+// @ts-ignore Could not find module lighthouse
+const PageDependencyGraph = require('lighthouse/lighthouse-core/computed/page-dependency-graph');
+const {auditNotApplicable} = require('../utils/builder');
+const {Audit} = require('lighthouse');
+const {getPageStartTime} = require('../utils/network-timing');
+const {getTransitiveClosure} = require('../utils/graph');
+const {isGptAdRequest} = require('../utils/resource-classification');
+
+/**
+ * Table headings for audits details sections.
+ * @type {LH.Audit.Details.Table['headings']}
+ */
+const HEADINGS = [
+  {
+    key: 'request',
+    itemType: 'url',
+    text: 'Request',
+  },
+  {
+    key: 'lineNumber',
+    itemType: 'numeric',
+    text: 'HTML Line Number',
```

Comment:
I don't love the phrasing of "HTML Line Number", what would you think instead of calling this column "Location" and using `.../pathname:nnn`.

Reply:
I removed line number since I think it's the line number in the script, not HTML. I'll look into it more as a follow up